packages/lit-dev-content/site/tutorials/content/word-viewer/00.md

```diff
@@ -0,0 +1,61 @@
+In this tutorial you will invent your own HTML element: `<word-viewer>`.
+You can see it on the right!
+
+The `word-viewer` component takes a list of words separated by periods, and
+shows them one at a time. You use it in HTML like this:
+
+```html
+<word-viewer words="Hello.World"></word-viewer>
+```
+
+Clicking it plays the words in reverse, and changes some styles.
+
+<litdev-aside type="info">
+
+Check out the video that accompanies this tutorial!
+
+[![How to build your first Lit component Youtube
+thumbnail](/images/tutorials/word-viewer/youtube_thumbnail.jpeg)](https://www.youtube.com/watch?v=QBa1_QQnRcs)
```

Comment:
No action needed but the thumbnail takes up a lot of space for something that's inside an aside. Since there's one at the last step, maybe this could just be a link? Or set a max-width? I am unsure what method of that works with our markdown parser. `<img>` tag with `width` attribute, or `style="mad-width:xxx"`? Totally okay to leave as is though.

Reply:
I vote for a:

```md
## Check out the video that accompanies this tutorial

[![How to build your first Lit component Youtube
thumbnail](/images/tutorials/word-viewer/youtube_thumbnail.jpeg)](https://www.youtube.com/watch?v=QBa1_QQnRcs)
```

Reply:
Done